adapter-manager.js

```diff
@@ -306,13 +306,25 @@ class AdapterManager extends EventEmitter {
     for (var adapterName in config.adapters) {
       var adapterConfig = config.adapters[adapterName];
 
-      if (adapterConfig.enabled) {
+      var loadAdapter = adapterConfig.enabled;
+      var notLoadReason = 'disabled';
+      if (process.env.NODE_ENV === 'test') {
```

Comment:
I like the idea of having a testing config versus a regular config. I'll take a stab at consolidating the config into a single file. There is a bunch of stuff in the config that isn't really config, but rather constants (like /things, etc), so I think I'll split those out of config too,